pkg/sql/sqlbase/privilege.go

```diff
@@ -78,6 +78,12 @@ func (p *PrivilegeDescriptor) removeUser(user string) {
 	p.Users = append(p.Users[:idx], p.Users[idx+1:]...)
 }
 
+// NewCustomRootPrivilegeDescriptor returns a privilege descriptor for the root user
```

Comment:
The reason for this one it to make it easier to have another `NewCustomAdminPrivilegeDescriptor` which will include both the `root` user and `admin` role once it exists.